concepts/functions/about.md

```diff
@@ -19,6 +22,55 @@ func PrintHelloName(name string) {
 }
 ```
 
+Parameters of the same type can be declared together, followed by a single type declaration.
+
+```go
+import "fmt"
+
+func PrintGreetingName(greeting, name string) {
+  fmt.Println(greeting + " " + name)
+}
+```
+
+## Parameters vs. Arguments
+
+Let's quickly cover two terms that are often confused together: `parameters` and `arguments`. 
+Function parameters are the names defined in the function's signature, such as `greeting` and `name` in the function `PrintGreetingName` above.
+Function arguments are the concrete values passed to the function parameters when we invoke the function.
+For instance, in the example below, `"Hello"` and `"Katrina"` are the arguments passed to the `greeting` and `name` parameters:
+
+```go
+PrintGreetingName("Hello", "Katrina")
+```
+
+## Pass by Value & Pass by Pointer
+
+It is also important to clarify the concept of passing by value as well as passing by pointer (sometimes referred to as _by reference_).
+In the example below, `name`, the `string` parameter to the function `PrintHelloName`, is passed into the function by value.
+This means that Go essentially makes a copy of the original value so that only this copy is accessed or modified by our function.
+
+```go
+import "fmt"
+
+func PrintHelloName(name string) {
+  fmt.Println("Hello " + name)
+}
+```
+
+Strictly speaking, all arguments are passed by value in Go, i.e. a copy is made of the value or data provided to the function.
+But if you don't want to make a copy of the data that you choose to pass to a function and
+want to actually access and change the data in the function,
+then you should use pointers as arguments, a.k.a. pass by reference.
```

Comment:
I feel like this explanation deserves an example. We give an example later in the text in the "Pointer" section which has some overlap with this one.

My suggestion here would be to join this explanation with the explanation in the "Pointers" section and just have one section talking about this. And maybe this explanation should be at the end of the document.

If you still think it makes sense to have to have 2 separate sections, I think they should at least be together and not separated like they are now.